build.rs

```diff
@@ -20,15 +20,29 @@ fn link_cuda() {
     #[cfg(feature = "driver")]
     println!("cargo:rustc-link-lib=dylib=cuda");
     #[cfg(feature = "driver")]
-    println!("cargo:rustc-link-lib=dylib=cudart");
+    println!("cargo:rustc-link-lib=static=cudart_static");
```

Comment:
Didn't realize until this PR, but we can actually remove this lib all together - we don't use any runtime apis